test/routers/healthcheck.unit.ts

```diff
@@ -0,0 +1,12 @@
+import app from "../../src/app";
+import request from "supertest";
+import { PrefixedUrls } from "../../src/constants";
+
+describe("healthcheck tests", () => {
+    it("should show status 200", async () => {
+        const resp = await request(app).get(PrefixedUrls.HEALTHCHECK);
```

Comment:
We could indicate test is not ready by skipping it

```suggestion
    it.skip("should show status 200", async () => {
```